internal/controller/assets/daemonset.go

```diff
@@ -336,7 +249,17 @@ func Daemonset(dsName, image, serviceAccount string, node *falconv1alpha1.Falcon
 							Resources: dsResources(node),
 						},
 					},
-					Volumes:           volumes(node),
+					Volumes: []corev1.Volume{
```

Comment:
nit: Why not keep the existing function `volumes` and update `volumes` to return `falconstore`? It would help with readability.